crates/biome_html_parser/src/syntax/mod.rs

```diff
@@ -312,6 +312,13 @@ fn parse_attribute(p: &mut HtmlParser) -> ParsedSyntax {
             .ok();
 
         Present(m.complete(p, HTML_ATTRIBUTE))
+    } else if p.at(T!['{']) {
+        m.abandon(p);
+        HtmlSyntaxFeatures::SingleTextExpressions.parse_exclusive_syntax(
+            p,
+            |p| parse_single_text_expression(p, HtmlLexContext::InsideTag),
+            |p: &HtmlParser<'_>, m: &CompletedMarker| disabled_interpolation(p, m.range(p)),
```

Comment:
The error isn't correct. The interpolation is for `{{ }}` syntax. I think we need a new error for Svelte

Reply:
Yeah I believe so. For now, single bracket expressions are used only for svelte files